.travis.yml

```diff
@@ -1,16 +1,44 @@
 language: go
 go:
-  - 1.7.x
+  - "1.7.x"
 dist: trusty
 sudo: required
 group: edge
+
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install -y autoconf pkg-config e2fslibs-dev libblkid-dev zlib1g-dev liblzo2-dev asciidoc
-install: ./scripts/travis.sh install
+  - sudo apt-get install -y python-apt autoconf pkg-config e2fslibs-dev libblkid-dev zlib1g-dev liblzo2-dev asciidoc
+install:
+  - sudo pip install ansible==2.3.1 docker-py
+  - ./scripts/travis.sh install
+before_script:
+  - tmp=`mktemp`
+  - echo '{"insecure-registries":["172.30.0.0/16"]}' > ${tmp}
+  - sudo mv ${tmp} /etc/docker/daemon.json
+  - sudo mount --make-shared /
+  - sudo service docker restart
 script:
   - ./scripts/travis.sh format
   - ./scripts/travis.sh vet
   - ./scripts/travis.sh lint
   - ./scripts/travis.sh build
   - ./scripts/travis.sh test
+  - ./scripts/travis.sh ci
+  - sudo docker pull docker.io/openshift/origin:v3.6.0-rc.0
+  - oc cluster up --image=docker.io/openshift/origin --version=v3.6.0-rc.0 --service-catalog=true
```

Comment:
I'm going to add back in the routing_suffix and try and get that working. It was failing to work before so I moved on, but it looks like we're going to need it.